src/components/Paginate.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useEffect, useState } from "react";
+
+const getPageCount = (
+  numberOfResults: number,
+  numberOfResultsPerPage: number
+): number => Math.ceil(numberOfResults / numberOfResultsPerPage);
+
+interface paginateProps {
+  children: any; // children: component?
```

Comment:
`any` is an antipattern. Better to use specific type here.

Reply:
bd6dc53 fixes this.